packages/runner/README.md

```diff
@@ -8,56 +8,106 @@ Run scanning for vulnerabilities just from your unit tests on CI phase.
-| ---------------------- | --------- | ----------------------------- | -------------------------------------------------------------------------------------------------------------------------------------- |
-| `tests`                | Yes       | N/A                           | The list of tests to be performed against the target application. [See full list](https://docs.brightsec.com/docs/vulnerability-guide) |
-| `smart`                | No        | true                          | Enabled by default to minimize scan time by using automatic smart decisions regarding parameter skipping, detection phases, etc.       |
-| `skipStaticParams`     | No        | true                          | Detects if a parameter has any effect on the target behavior when changed, and skip testing static parameters.                         |
-| `poolSize`             | No        | 10                            | Sets the maximum concurrent requests for the scan, to control the load on your server.                                                 |
-| `attackParamLocations` | No        | ['body', 'query', 'fragment'] | Defines which part of the request to attack. Additionally available values are 'header' and 'path'                                     |
-| `slowEpTimeout`        | No        | 1000                          | Skip entry-points that take longer to respond than specified ms value                                                                  |
-| `name`                 | No        | _endpoint name_               | The scan name. Default one will look like `GET https://example.com/`.                                                                  |
+In general, each found issue will cause the scan to stop. To control this behavior you can set a severity threshold using the `threshold` method:
 
-</details>
+```ts
+scan.threshold(Severity.HIGH);
+```
 
-4. Run `SecScan`:
+Now found issues with severity lower than `HIGH` will not cause the scan to stop.
 
-   - call `run(target: TargetOptions)` method of `SecScan` instance. Scanning is performed via [repeater](https://github.com/NeuraLegion/sec-tester-js/tree/master/packages/repeater), that is created automatically behind the scene.
+Due to configuration issues, let's say the scan might take much more time than you expect. In this case, you can provide a timeout (in milliseconds) for specifying how long to wait before aborting.
 
-     - Returns promise that is resolved if scan finishes w/o any vulnerability found, and is rejected otherwise (on founding issue that meets threshold, on timeout, on scanning error)
+```ts
+scan.timeout(30000);
+```
 
-     - If any vulnerabilities are found - they will be pretty printed to stdout or stderr (depending on severity) by [reporter](https://github.com/NeuraLegion/sec-tester-js/tree/master/packages/reporter).
+After 30 seconds, if the scan isn't finishing or finding any vulnerability, it will throw an error.
```

Comment:
please consider changing to 

```
Sometimes either due to scan configuration issues or target misbehave, the scan might take much more time than you expect. In this case, you can provide a timeout (in milliseconds) for specifying maximum scan running time.
```

and 

```
In that case after 30 seconds, if the scan isn't finishing or finding any vulnerability, it will throw an error.

```